ledger/accountdb_test.go

```diff
@@ -3313,3 +3323,39 @@ func TestAccountOnlineAccountsNewRound(t *testing.T) {
 	_, _, err = onlineAccountsNewRoundImpl(writer, updates, proto, lastUpdateRound)
 	require.Error(t, err)
 }
+
+func TestAccountOnlineRoundParams(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
+	proto := config.Consensus[protocol.ConsensusCurrentVersion]
+
+	dbs, _ := dbOpenTest(t, true)
+	setDbLogging(t, dbs)
+	defer dbs.Close()
+
+	tx, err := dbs.Wdb.Handle.Begin()
+	require.NoError(t, err)
+	defer tx.Rollback()
+
+	var accts map[basics.Address]basics.AccountData
+	accountsInitTest(t, tx, accts, protocol.ConsensusCurrentVersion)
+
+	onlineRoundParams := make([]ledgercore.OnlineRoundParamsData, 80+proto.MaxBalLookback)
+	for i := range onlineRoundParams {
+		onlineRoundParams[i].OnlineSupply = uint64(i)
+	}
+
+	err = accountsPutOnlineRoundParams(tx, onlineRoundParams, 1)
```

Comment:
if set rnd=0 then there will be no 10 and 9 below, only 10.

Reply:
onlineRoundParams[9] corresponds to round 10 yes. This is a bit confusing, should probably make more clear.